site/src/component/Review/SubReview.tsx

```diff
@@ -91,6 +96,11 @@ const SubReview: FC<SubReviewProps> = ({ review, course, professor, colors, colo
 
   return (
     <div className="subreview">
+      {editable && editReview && (
+        <div className="edit-pen-icon" onClick={() => editReview(review, course, professor)}>
+          <FaPen />
+        </div>
```

Comment:
Should make this a button not a div for accessibility purposes and then style it appropriately. I agree with Ethan's comments on the button styling.